app/docs/src/how-tos/aws-eks.md

```diff
@@ -0,0 +1,520 @@
+```
+
+The result is something similar to:
+
+```
+aws eks update-kubeconfig --name demo-eks-cluster --region us-east-1
+Added new context arn:aws:eks:us-east-1:573113295293:cluster/demo-eks-cluster to /Users/stack72/.kube/config
+```
+
+Using [kubectl](https://kubernetes.io/docs/reference/kubectl/), you can check
+the access permissions to the cluster:
+
+```
+kubectl auth can-i get pods
+```
+
+The result may be something similar to:
+
+```
+E1112 00:25:56.469718   29444 memcache.go:265] couldn't get current server API group list: the server has asked for the client to provide credentials
+E1112 00:25:57.309084   29444 memcache.go:265] couldn't get current server API group list: the server has asked for the client to provide credentials
+Warning: the server doesn't have a resource type 'pods'
+
+error: You must be logged in to the server (Unauthorized)
+```
+
+This is because my local user isn't part of the Cluster access configuration
+
+Start in a change set named `EKS Cluster Access`.
+
+### Update EKS Cluster Access Config
+
+![Update Cluster Config](./aws-eks/update-cluster-config.png)
+
+Update the `demo-eks-cluster` `accessConfig.authenticationMode` to be
+`API_AND_CONFIG_MAP`
+
+Right click on the component and select `Update Cluster Access Config`.
+
+Note: This change needs to take effect in the cluster before you can add access
+policies. So you will need to retry the failed access entry actions.
+
+### Create Access Entry
+
+![Create Access Entry](./aws-eks/create-access-entry.png)
+
+Add an `Access Entry` to your `demo-eks-cluster` eks cluster frame.
+
+Set the component name to be `My User`.
+
+Set the `type` to be `STANDARD`.
+
+Set the `principalArn` to be `<enter your userArn>`.
+
+Add 2 `Access Policy` components to your `demo-eks-cluster` eks cluster frame.
+
+Configure them to be as follows:
+
+| Component Name              | `policyArn`                                                        | `accessScope.type` |
+| --------------------------- | ------------------------------------------------------------------ | ------------------ |
+| AmazonEKSAdminPolicy        | arn:aws:eks::aws:cluster-access-policy/AmazonEKSAdminPolicy        | cluster            |
+| AmazonEKSClusterAdminPolicy | arn:aws:eks::aws:cluster-access-policy/AmazonEKSClusterAdminPolicy | cluster            |
+
+Connect the `Principal ARN` output socket of the `My User` component to the
+`Principal ARN` input sockets of the Access Policy components
+
+### Apply your Change Set
+
+Press `Escape` or click anywhere on the canvas background to select the
+Workspace.
+
+Click the `Apply Change Set` button to:
+
+- Create the correct access policies for your user
+
+NOTE: By adding a user to the Cluster access management, you will need to
+regenerate a new kubeconfig.
+
+### Connect to the Cluster
+
+After generating an updated Kubeconfig, you can run the command:
+
+```
+$ kubectl auth can-i get pods
+yes
+```
+
+Now you can access the details of the pods:
```

Comment:
The first line here isn't rendering + looks a bit confusing. Basically it doesn't show the command to run, you can just copy it out of the terminal